ENIGMAsystem/SHELL/Graphics_Systems/OpenGLES2/Makefile

```
@@ -1,2 +1,9 @@
 include Graphics_Systems/OpenGLES/Makefile
-SOURCES += $(wildcard Graphics_Systems/OpenGLES2/*.cpp)
+
+override CXXFLAGS += -I$(ENIGMA_ROOT)/android/external/gles2/include
+override CFLAGS += -I$(ENIGMA_ROOT)/android/external/gles2/include
+SOURCES += $(wildcard Graphics_Systems/OpenGLES2/*.cpp) $(ENIGMA_ROOT)/android/external/gles2/src/gles2.c
+
+ifeq ($(TARGET-PLATFORM), Android)
+	ANDROID_LDLIBS += -lGLESv2
+endif
```

Shouldn't these be moved down inside the `TARGET-PLATFORM`/`Android` check?
```suggestion
ifeq ($(TARGET-PLATFORM), Android)
	override CXXFLAGS += -I$(ENIGMA_ROOT)/android/external/gles2/include
	override CFLAGS += -I$(ENIGMA_ROOT)/android/external/gles2/include
	SOURCES += $(wildcard Graphics_Systems/OpenGLES2/*.cpp) $(ENIGMA_ROOT)/android/external/gles2/src/gles2.c
	ANDROID_LDLIBS += -lGLESv2
endif
```